MyoSimulatorForm/MyoSimGUITest/Class1.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace MyoSimGUITest
+{
+    [TestFixture]
+    public class MyoSimTestClass
+    {
+        [TestCase]
+        public void AddPose()
+        {
+
+        }
+    }
```

Comment:
Does this file need to exist? If not, please delete. If so, please rename the file. Class1.cs is clearly a default.